lua/inc/com/centreon/broker/lua/macro_cache.hh

```diff
@@ -58,9 +68,14 @@ namespace         lua {
 
     void           _process_instance(instance_broadcast const& in);
```

Comment:
The __process_instance()_ method should clean cache. Will this be available in another PR ?

Reply:
So this is also an issue in influxdb code...

Reply:
Yes indeed the issue is in InfluxDB and Graphite modules too.

When receiving an _instance_broadcast_ event, all items *related to this instance* should be discarded. This means that hash tables should not be cleared entirely but instead objects related to the target instance should be removed. With the current data structure this will likely be inefficient though.